packages/mobile/src/config/routing/NavigationName.ts

```diff
@@ -8,6 +8,7 @@ export enum NavigationName {
 
     MENU = 'Menu',
     BOTTOMMENU = 'BottomMenu',
+    DRAWER = 'DrawerNavigation',
```

Comment:
To nie jest nazwa ekranu. Czy powinien ten twór tu być? i czy musi istniec aby nawigacja dzialala?

Reply:
naprawione